contracts/HydraDelegation/Delegation.sol

```diff
@@ -80,8 +82,25 @@ contract Delegation is
     /**
      * @inheritdoc IDelegation
      */
-    function setCommission(uint256 newCommission) external {
-        _setCommission(msg.sender, newCommission);
+    function setPendingCommission(uint256 newCommission) external {
+        _setPendingCommission(msg.sender, newCommission);
+    }
+
+    /**
+     * @inheritdoc IDelegation
+     */
+    function applyPendingCommission() external {
+        _applyPendingCommission(msg.sender);
+    }
+
+    /**
+     * @inheritdoc IDelegation
+     */
+    function setInitialCommission(uint256 initialCommission) external {
```

Comment:
I think the only option to set initial commission must be on register, otherwise a staker can start with zero commission and can then immediately set 100%

Reply:
faster implementation option is giving a gratis period of 2 hours for example for setting an initial commission without a wait period